package.json

```diff
@@ -14,49 +14,49 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "6.0.9",
-    "@angular/cdk": "6.3.3",
-    "@angular/common": "6.0.9",
-    "@angular/compiler": "6.0.9",
-    "@angular/core": "6.0.9",
-    "@angular/flex-layout": "6.0.0-beta.16",
-    "@angular/forms": "6.0.9",
-    "@angular/http": "6.0.9",
-    "@angular/material": "6.3.3",
-    "@angular/platform-browser": "6.0.9",
-    "@angular/platform-browser-dynamic": "6.0.9",
-    "@angular/router": "6.0.9",
-    "chart.js": "2.7.2",
-    "core-js": "2.4.1",
+    "@angular/animations": "^9.1.1",
```

Comment:
We had locked this down to specific versions of each package before because having the `^` or `~` seemed to randomly have issues with breaking package updates. Not sure if we want to do that or not. @roblouie

Reply:
I change the package json file to include only exact file version.  Developers can manually update the packages as needed.